src/main/java/emu/grasscutter/scripts/ScriptLib.java

```diff
@@ -727,17 +740,17 @@ public int SetGroupReplaceable(int groupId, boolean value) {
         return 1;
     }
 
-    public int[] GetSceneUidList(){
+    public LuaTable GetSceneUidList(){
```

Comment:
It seems to mostly use a for worth pairs for this, which seems to work, but yeah, probaly better to start with 1, in case they directly access 1 somewhere.